app.js

```diff
@@ -34,26 +33,50 @@ function lockCards(cardsToLock) {
 		card.removeEventListener('click', handleUserInput);
 	});
 	firstCard = null;
+	pausePlay = false;
+
 	console.log('someone called lockCards,Cards match');
-};
+}
 
 function handleUserInput() {
-	this.classList.add('turned'); // you always want to do this.
+	if (pausePlay) {
+		return;
+	}
+	this.classList.add('turned');
```

Comment:
you should encapsulate this in a function, `turnCardOver` and when you remove the class, call it `hideCard` or something that makes sense in English :D.

Reply:
I can get the `freezeCard` to work but `turnCard`, not so much. Not sure how I should do this.